bin/picca_deltas.py

```diff
@@ -613,18 +613,27 @@ def main(cmdargs):
 
     ### Mask BALs
     if not args.bal_catalog is None:
-        userprint("INFO: Adding BALs found in BAL catalog")
-        bal_cat = bal_tools.read_bal(args.bal_catalog)
+        userprint("INFO: Masking BALs from BAL catalog")
+        bal_cat = bal_tools.read_bal(args.bal_catalog,'EBOSS')
         num_bal = 0
         for healpix in data:
             for forest in data[healpix]:
                 if forest.thingid in bal_cat["THING_ID"]:
-                    mask_obs_frame_bal = []
-                    mask_rest_frame_bal = bal_tools.add_bal_rest_frame(
-                        bal_cat, forest.thingid, args.bal_index)
-                    forest.mask(mask_rest_frame_bal)
+                    bal_mask = bal_tools.add_bal_mask(
+                        bal_cat, forest.thingid, args.bal_index, 'THING_ID')
+                    forest.mask(bal_mask)
                     num_bal += 1
         log_file.write("Found {} BAL quasars in forests\n".format(num_bal))
+    elif (args.keep_bal is True) & (args.bal_catalog is None):
```

Comment:
as this is already an `elif` on the previous `if not args.bal_catalog is None`, the second half is superfluous.